Library/Styles/Colors.swift

```diff
@@ -208,6 +212,11 @@ extension UIColor {
     return .hex(0x282828)
   }
 
+  /// 0x1C1C1C
+  public static var ksr_text_black: UIColor {
```

Comment:
Is this different from `soft_black`? Just curious 👀

Reply:
yep. `text_black` is something I dug up from the pattern library for black text. It'd be good to start using it—especially where text is on white backgrounds—as `soft_black` can be little too light for legibility.